scripts/authority_keys.py

```diff
@@ -1,93 +1,100 @@
 import dbs
+import bson
 
 def convert_authority_keys():
     golos_authority_keys = dbs.golos_db['account_authority_object']
+    cyberway_permusage = dbs.cyberway_db['permusage']
+    cyberway_permission = dbs.cyberway_db['permission']
 
-    id_count = 0
+    id_count_permusage = cyberway_permusage.find().sort('id', -1).limit(1)[0]['id'] + 1
+    id_count_permission = cyberway_permission.find().sort('id', -1).limit(1)[0]['id'] + 1
 
     for doc in golos_authority_keys.find():
         permusage = {
-	    "id" : id_count,
+	    "id" : bson.Int64(id_count_permusage),
 	    "last_used" : "",
 	    "_SCOPE_" : "_CYBERWAY_",
 	    "_PAYER_" : "",
-	    "_SIZE_" : 16
+	    "_SIZE_" : bson.Int64(16)
         }
         dbs.cyberway_db['permusage'].save(permusage)
 
         active_permission = {
-	    "id" : id_count,
+	    "id" : bson.Int64(id_count_permission),
 	    "usage_id" : permusage["id"],
-	    "parent" : 0,
+	    "parent" : bson.Int64(0),
```

Comment:
For active & posting permission it's need to specify parent authority. So order for insert authority must be: owner, active, posting. And field parent must contain `id` from previous authority.
Try to execute `cleos get account <name>` for acounts from golos. You should get something like this:
```
/mnt/working/eos-debug.git$ cleos get account user1
created: 2018-10-24T10:57:42.500
permissions: 
     owner     1:    1 GLS7wFkt9bAQczy2JzsF9cJ2qCNKjKsHow7DhbzA4J6gu3qUpj5bU
        active     1:    1 GLS7wFkt9bAQczy2JzsF9cJ2qCNKjKsHow7DhbzA4J6gu3qUpj5bU
           posting     1:    1 GLS5EVPdg9xZkzTeSN5gE9PVcUjMd4F4XUfHePNnR7rdKQq6Qw6Ho
```